setup.py

```diff
@@ -29,6 +29,8 @@
     'dpdispatcher>=0.3.11',
     'netCDF4',
     'dargs>=0.2.9',
+    'pybind11>=2.4',
```

Comment:
@HuangJiameng I didn't get errors when I tried to install `pymatgen-analysis-defects`. Please check again.